README.md

```diff
@@ -1,7 +1,7 @@
 site_master
 ===========
 
-[![Build Status](https://travis-ci.org/UNLSiteMaster/site_master.svg?branch=master)](https://travis-ci.org/UNLSiteMaster/site_master)
+[![Build Status](https://travis-ci.org/CuBoulder/site_master.svg?branch=master)](https://travis-ci.org/CuBoulder/site_master)
```

Comment:
The README.md should be added to the .gitignore if we are going to customize it.  I don't think this needs to change as long as the PRs in the forked repo are running tests before a PR is made to parent repo.